uint8 PUMP_COMMAND_HOLD=0
uint8 PUMP_COMMAND_FILL=1
uint8 PUMP_COMMAND_EMPTY=2
uint8 command
uint8 pulse_time
